### DISCUSSION TITLE
Get Media Posts

### DISCUSSION THREAD

**Discussion by MariaJsn** [sbertix/Swiftagram#155] [Q&A] [answered]
Hi Sbertix,

I updated the library to latest version and Endpoint.Media.Posts.owned api does not return data. (Before it was)
It says 'Result of call to 'task(maxLength:by:onComplete:onChange:)' is unused'

What is wrong in my code?

My Code:

        Endpoint.Media.Posts.owned(by: self.secret.identifier)
        .unlocking(with: self.secret)
            .task(maxLength: 1, onComplete: { _ in
              
               
                
            }, onChange: { _ in
               
            })

Thanks,
Maria

**Comment by sbertix** [comment, marked as answer]
Hey @mariajsn

You simply need to resume the task 😊
Nothing has changed, you simply seem to have forgotten one line in your code. 

```swift
Endpoint.Media.Posts.owned(by: self.secret.identifier)
    .unlocking(with: self.secret)
    .task(maxLength: 1, onComplete: { _ in
          
           
            
        }, onChange: { _ in
           
        })
    .resume() // Add this. 
```
